packages/ui5-application-writer/src/index.ts

```diff
@@ -126,4 +126,4 @@ async function enableTypescript(basePath: string, fs?: Editor): Promise<Editor>
 }
 
 export { Ui5App, generate, enableTypescript, isTypescriptEnabled };
-export { App, Package, UI5, AppOptions };
+export { App, Package, UI5, AppOptions, UI5_DEFAULT };
```

Comment:
Do we need the new export here? If I didn't miss anything then you only import it in a test and there you refer to the source file directly.

Reply:
Removed in https://github.com/SAP/open-ux-tools/pull/845/commits/30c4337bcd33c2dff5e3e8ae4abd167fe26be480